Rakefile

```diff
@@ -67,7 +71,11 @@ task :sync do
   create.each do |issue|
     puts "+ Creating issue: #{issue[:title]}"
     body = ERB.new(File.read('_issue_description.erb')).result(binding)
-    client.create_issue(repo, issue[:title], body, :labels => "hacktoberfest,#{issue[:component]}")
+    labels = ['hacktoberfest']
+    labels << HTMLEntities.new.decode(issue[:component]) unless issue[:component].nil?
```

Comment:
The `HTMLEntities` business is here because I noticed at least one issue where a label contained the hard coded string `&amp;`. So this fixes that small issue.

Reply:
YES